appveyor.yml

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
I can do that. Doesn't look like Appveyor has Bazel installed in its images. It would be straight forward enough to add, though as it is in [chocolatey](https://chocolatey.org/packages/bazel). Additionally, the developers are pretty good about installing software into their images if you open a support ticket. 

I don't see Bazel being used in this project though, or can it be used with Maven? I've never used Bazel.